openpifpaf/datasets/factory.py

```diff
@@ -1,17 +1,58 @@
 import torch
 
 from .coco import Coco
+from .visdrone import VisDrone
+from .uavdt import UAVDT
+from .eurocity import EuroCity
+from .nightowls import NightOwls
+from .visual_relationship import VisualRelationship
 from .collate import collate_images_targets_meta
-from .constants import COCO_KEYPOINTS, HFLIP
+from .constants import COCO_KEYPOINTS, HFLIP, BBOX_KEYPOINTS, BBOX_HFLIP
 from .. import transforms
 
-COCOKP_ANNOTATIONS_TRAIN = 'data-mscoco/annotations/person_keypoints_train2017.json'
```

Comment:
I was wondering about this. Since it will support other datasets, I changed this so that there a "data" folder with the different datasets in it. Should I keep the mscoco in the old folder structure and the other datasets in the folder structure I proposed?